build.gradle

```diff
@@ -18,14 +17,48 @@ task buildDocumentation(type: Exec) {
     args '-c', "docker run --volume $rootDir/docs:/docs --rm squidfunk/mkdocs-material:5.2.2 build"
 }
 
+static getCustomPublicationName(project) {
+    if (project.name == 'gradle-plugin') {
+        return 'pluginMaven'
```

Comment:
why is this different than the artifact ID? can't it be called `proguard-gradle`?

Reply:
The `proguard-plugin` project uses the `java-gradle-plugin` plugin, which means that after the configuration phase the project will hold a publication with the fixed name `pluginMaven`. The use of the plugin was added after I created the publication logic which means that the project currently had 2 distinct publications. It is rather lucky that we still push artifacts manually from bintray to JCenter so the extra publication did not end up in the public space.

The next forced pushed makes the logic slightly more generic with added comments.